asreview/webapp/api.py

```diff
@@ -1252,9 +1268,16 @@ def api_get_progress_density(project):
     include_priors = request.args.get('priors', False, type=bool)
 
     try:
+        project_config = project.config
```

Comment:
`project.config` is optimized in many ways. There is no need to make a copy :) 

The next thing is perfectly fine:

```
            if project.config['reviews'][0]['status'] == 'finished' \
                    and project.config['mode'] == PROJECT_MODE_SIMULATE:
```